### ISSUE TITLE
Analysis Area Workflow fails on regions step

### ISSUE THREAD

**Issue by dwuthrich** [archesproject/arches-for-science-prj#992]
The analysis area workflow throws an error on load of the regions step (as shown in attached image) and hangs on the "updating annotations" step when saving an annotation area![Screen Shot 2022-09-26 at 1.46.25 PM.png](https://images.zenhubusercontent.com/56f1ad0a6caf56c9d38e1cbd/137ef187-5852-484e-afa6-fcd24bd7b8bf)

**Comment by dwuthrich** [comment]
The sample taking workflow fails in a similar fashion

**Comment by njkim** [comment]
@dwuthrich 
I believe it's been resolved in archesproject/arches#8962 but it requires migration back and forth.
Alternatively, running the following script in Postgres should fix the issue.
Let me know if it doesn't.

```
DROP VIEW IF EXISTS vw_annotations;

CREATE OR REPLACE VIEW vw_annotations AS
SELECT uuid(json_array_elements(t.tiledata::json->n.nodeid::text->'features')->>'id') AS feature_id,
    t.tileid,
    t.tiledata,
    t.resourceinstanceid,
    t.nodegroupid,
    n.nodeid,
    jsonb_array_elements(t.tiledata::jsonb->n.nodeid::text->'features') AS feature,
    json_array_elements(t.tiledata::json->n.nodeid::text->'features')->'properties'->>'canvas' AS canvas
FROM tiles t
    LEFT JOIN nodes n ON t.nodegroupid = n.nodegroupid
WHERE (
        (
            SELECT count(*) AS count
            FROM jsonb_object_keys(t.tiledata) jsonb_object_keys(jsonb_object_keys)
            WHERE (
                    jsonb_object_keys.jsonb_object_keys IN (
                        SELECT n_1.nodeid::text AS nodeid
                        FROM nodes n_1
                        WHERE n_1.datatype = 'annotation'::text
                    )
                )
        )
    ) > 0
AND n.datatype = 'annotation'::text;
```

**Comment by dwuthrich** [comment]
@njkim Thanks, that solved the issue.  I'll close this ticket
